core/src/trezor/ui/layouts/tt_v2/__init__.py

```diff
@@ -419,8 +417,8 @@ async def confirm_output(
     color_to: int = ui.FG,  # TODO cleanup @ redesign
     to_str: str = " to\n",  # TODO cleanup @ redesign
     to_paginated: bool = False,  # TODO cleanup @ redesign
-    width: int = MONO_ADDR_PER_LINE,
-    width_paginated: int = MONO_ADDR_PER_LINE - 1,
+    width: int = 0,  # TODO cleanup @ redesign
```

Comment:
what's this about?

Reply:
The constants are imported from `...constants.tt` which we don't have in the frozen build.